resource/ui/ts/models/status.ts

```diff
@@ -0,0 +1,218 @@
+// source: models/store_status.ts
+/// <reference path="../typings/mithriljs/mithril.d.ts" />
+/// <reference path="../typings/d3/d3.d.ts" />
+/// <reference path="stats.ts" />
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
+// Author: Matt Tracy (matt@cockroachlabs.com)
+
+module Models {
+    export module Status {
+        import promise = _mithril.MithrilPromise;
```

Comment:
Should you have a sub module here for node and store instead of having them in the same module?